[2022-08-03 13:50:15,594] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T13:40:00+00:00 [queued]>
[2022-08-03 13:50:15,824] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T13:40:00+00:00 [queued]>
[2022-08-03 13:50:15,825] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 13:50:15,826] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-03 13:50:15,827] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 13:50:15,889] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-03T13:40:00+00:00
[2022-08-03 13:50:15,903] {standard_task_runner.py:51} INFO - Started process 18983 to run task
[2022-08-03 13:50:15,916] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'my_simple_dag', 'respond', '2022-08-03T13:40:00+00:00', '--job-id', '1134', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_simple_dag.py', '--cfg-path', '/tmp/tmponvroj41']
[2022-08-03 13:50:15,929] {standard_task_runner.py:76} INFO - Job 1134: Subtask respond
[2022-08-03 13:50:16,111] {logging_mixin.py:103} INFO - Running <TaskInstance: my_simple_dag.respond 2022-08-03T13:40:00+00:00 [running]> on host b4dcda255553
[2022-08-03 13:50:16,261] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T13:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-03T13:40:00+00:00
[2022-08-03 13:50:16,264] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-03 13:50:16,620] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=my_simple_dag, task_id=respond, execution_date=20220803T134000, start_date=20220803T135015, end_date=20220803T135016
[2022-08-03 13:50:16,751] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-03 13:50:16,809] {local_task_job.py:118} INFO - Task exited with return code 0
